lib/wahwah/helper.rb

```diff
@@ -62,7 +62,7 @@ def self.file_format_from_signature(io)
       # M4A is checked for first, since MP4 files start with a chunk size -
       # and that chunk size may incidentally match another signature.
       # No other formats would reasonably have "ftyp" as the next for bytes.
-      return "m4a" if ["ftypM4A ".b, "ftyp3gp4".b].include?(signature[4...12])
+      return "m4a" if ["ftypM4A ".b, "ftyp3gp4".b, "ftypmp42".b].include?(signature[4...12])
```

Comment:
Similar to the Samsung recorder issue from #47, an `ftypmp42` can also be opened as an "m4a" file.